tensorflow_blade/src/tensorrt/BUILD

```diff
@@ -57,6 +58,9 @@ cc_library(
         "//src/util:logging",
     ] + if_has_cublaslt([
         "@local_config_cuda//cuda:cublasLt",
+    ]) + if_has_myelin([
+        "@local_config_cuda_static//:nvrtc",
```

Comment:
why is nvrtc bind with myelin ?

Reply:
For trt7, if nvrtc is not linked when myelin libs are linked, we will encounter undefined symbol issue, this symbol is from nvrtc. However for later trt versions, such as trt8, myelin libs no longer exists, we don't need to link nvrtc